docker-images/python-36-oracle/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM "__FROM_IMAGE_STREAM_DEFINED_IN_TEMPLATE__"
+
+ENV LD_LIBRARY_PATH="${APP_ROOT}/oracle_bin/instantclient" \
+    OCI_HOME="$APP_ROOT/oracle_bin/instantclient" \
+    OCI_LIB_DIR="$APP_ROOT/oracle_bin/instantclient" \
+    OCI_INCLUDE_DIR="$APP_ROOT/oracle_bin/instantclient/sdk/include"
+
+RUN mkdir -p /tmp/oracle
+COPY oracle/ /tmp/oracle
+
+RUN mkdir -p ${APP_ROOT}/oracle_bin
+
+USER 0
+
+RUN yum -y install libaio
+
+# yum install removed for openshift build
+# pkgs build-essential libaio1 python-minimal - don't exist for RHEL 
+# pkgs curl unzip git - are all already installed on python RHEL image
+
+RUN unzip /tmp/oracle/instantclient-basic-linux.x64-19.3.0.0.0.zip -d ${APP_ROOT}/oracle_bin && \
```

Comment:
Have you seen any license information on these zip files and if we're allowed to copy them into github?